From: brian@murphy.dk
Subject: [PATCH] USB: set timeout message to debug level: message.c

[PATCH] USB: set timeout message to debug level: message.c

	this makes a the debug message on timeout be at debug level
so it only exists when compiling with explicit debugging support.
I have added information about the request length and data length
as now if the data length is non-zero a 0 status is returned.

Signed-off-by: Brian Murphy <brian@murphy.dk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/core/message.c |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)


diff -Nru a/drivers/usb/core/message.c b/drivers/usb/core/message.c
--- a/drivers/usb/core/message.c	2005-03-08 16:42:59 -08:00
+++ b/drivers/usb/core/message.c	2005-03-08 16:42:59 -08:00
@@ -65,11 +65,14 @@
 		status = urb->status;
 		/* note:  HCDs return ETIMEDOUT for other reasons too */
 		if (status == -ECONNRESET) {
-			dev_warn(&urb->dev->dev,
-				"%s timed out on ep%d%s\n",
+			dev_dbg(&urb->dev->dev,
+				"%s timed out on ep%d%s len=%d/%d\n",
 				current->comm,
 				usb_pipeendpoint(urb->pipe),
-				usb_pipein(urb->pipe) ? "in" : "out");
+				usb_pipein(urb->pipe) ? "in" : "out",
+				urb->actual_length,
+				urb->transfer_buffer_length
+				);
 			if (urb->actual_length > 0)
 				status = 0;
 			else
